tests/baselines/reference/unusedClassesinModule1.errors.txt

```diff
@@ -1,10 +1,10 @@
-tests/cases/compiler/unusedClassesinModule1.ts(2,11): error TS6133: 'Calculator' is declared but its value is never read.
+tests/cases/compiler/unusedClassesinModule1.ts(2,5): error TS6133: 'Calculator' is declared but its value is never read.
 
 
 ==== tests/cases/compiler/unusedClassesinModule1.ts (1 errors) ====
     module A {
         class Calculator {

```

Comment:
What's going to happen for generic class?

`Calculator<T>`
`Calculator<T where T extends K>`